### PR TITLE
#161837243 redirect user after verification

### PR DESCRIPTION
**Pull request by gloriaodipo**
## What does this PR do?
It implements redirect to login page after verification.

## Description of Task to be completed?
Once a user has verified their account from their email, they should be redirected back to the application to a page where they are greeted with a message telling them that they have successfully activated their account.

## How should this be manually tested?
Clone this branch ft-account-verification-161837243
Run the following commands on your terminal:
`yarn install` - to install required dependencies
`yarn start` - to start server
When the server starts, you will be redirected to the browser
Navigate to this url: [http://localhost:3000/signup](http://localhost:3000/signup) and enter details to create an account.
After successful signup as shown below; check your email for a verification email sent with a link containing a token to verify your account.

image

**Screenshot of email verification link:**  Click on the link in the email, this should take you to the login page.

image

## Any background context you want to provide?
N/A

## What are the relevant pivotal tracker stories?
[161837243](https://www.pivotaltracker.com/n/projects/2198533/stories/161837243)

## Deployment note
N/A

## Related PRs branch | PR (branch_name) | (pr_link)
[ft-account-verification-161837243](ft-account-verification-161837243)

## Todos
- [ ] Raise PR
- [ ] Test
- [ ] Documentation

### CODE REVIEW

File: src/reducers/verification.reducer.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import constants from '../constants/index';
+import initialState from './initialState';
+
+const { VERIFY } = constants;
+
+const verifyReducer = (state = initialState.signUp, action) => {
```
Comment (Arthur236) [inline src/reducers/verification.reducer.js]: Why not use the `verify` initial state here?

Comment (gloriaodipo) [reply src/reducers/verification.reducer.js]: It's because I've named my action creator "verify"

Comment (gloriaodipo) [reply src/reducers/verification.reducer.js]: Oh yeah, i'll change that right away

**Comment by Arthur236** [comment]
screenshot 2018-11-15 at 10 32 24

Hey @gloriaodipo
I dont think this is the page we should be redirecting the user to once they verify their email.
It should be a page within our app

File: src/__tests__/containers/VerificationContainer.test.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import { mapStateToProps } from '../../containers/Auth/VerificationContainer';
+import verifyReducer from '../../reducers/verification.reducer';
+
+it('correctly maps state to props', () => {
+  const verify = verifyReducer;
+  expect(mapStateToProps(verify)).toEqual({});
```
Comment (lukorito) [inline src/__tests__/containers/VerificationContainer.test.js:6]: You could use the method  `toHaveProperty`  to find the property in an object

**Review by Arthur236 — APPROVED** [review]
Looks good 👍

**Pull request merged**